docs/learn/learn-extrinsics.md

```diff
@@ -48,6 +48,25 @@ Here are some key differences between the different types of extrinsics:
 - Inherents exist to address the need of adding some data to a block, whereas signed or unsigned
   transactions exist to potentially change the state of the blockchain.
 
+### Mortal and Immortal Extrinsics
+
+Extrinsics can be mortal (i.e. valid within a defined block interval) or immortal (i.e. always
+valid). By default and for security reasons, all extrinsics submitted using the
+[Polkadot-JS UI](../general/polkadotjs-ui.md) and
+[browser extension](../general/polkadotjs.md#polkadot-js-extension) will be mortal. This will avoid
+the chance of being a victim of replay attacks after
+[reaping an account](./learn-accounts.md#existential-deposit-and-reaping).
+
+Reaping an account resets the nonce value. If all signed transactions until the nonce before the
```

Comment:
Add "Replay attacks work as follows:" or similar to the beginning of this paragraph.